windowing.py

```diff
@@ -70,17 +69,17 @@ def find_lift_windows(windowed_data, divisor):
     lift_windows = np.take(windows, indices, axis=0)
     return lift_windows
 
-# Input: windowed_data = np array
-# Input: threshold = int (variance threshold for start of lift
-#
-# Description: finds the more refined start time by calcualting variacne
-#				on the smaller windows and returning the start time of the first
-#				window that meets the thresholdusing a smaller threshold.
-#
-# Return: refined start time = int (in nanos)
-
 
 def refine_start_time(windowed_data, threshold):
+    # Description: finds the more refined start time by calcualting variacne
+    #				on the smaller windows and returning the start time of the first
+    #				window that meets the thresholdusing a smaller threshold.
+    #
+    # Input: windowed_data = np array
+    # Input: threshold = int (variance threshold for start of lift
+    #
+    #
+    # Return: refined start time = int (in nanos)
     windows = windowed_data[0]
```

Comment:
could you reuse code from find_lift_windows by using a subfunction?

Before reworking see comment in find_precise_start_time